src/core/service.c

```diff
@@ -2763,7 +2763,13 @@ static int service_serialize_exec_command(Unit *u, FILE *f, const ExecCommand *c
                 return log_oom();
 
         key = strjoina(type, "-command");
-        (void) serialize_item_format(f, key, "%s %u %s %s", service_exec_command_to_string(id), idx, p, args);
+        (void) serialize_item_format(
+                        f, key,
+                        "%s %s%u %s %s",
+                        service_exec_command_to_string(id),
+                        command->command_next ? "" : "+",
```

Comment:
this is mean. but maybe ok. but please add a comment explaining this hack here to the sources